ballerina/http_log_manager.bal

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+# Represents HTTP trace log configuration.
+#
+# + consoleEnabled - Boolean value to enable or disable console trace logs
+# + path - File path to store trace logs
+# + host - Socket hostname to publish the trace logs
+# + port - Socket port to publish the trace logs
+public type TraceLogConfiguration record {|
+    boolean consoleEnabled = false;
+    string path = "";
+    string host = "";
+    int port = 0;
+|};
+
+# Represents HTTP access log configuration.
+#
+# + consoleEnabled - Boolean value to enable or disable console access logs
+# + path - File path to store access logs
+public type AccessLogConfiguration record {|
+    boolean consoleEnabled = false;
+    string path = "";
+|};
+
+configurable TraceLogConfiguration & readonly traceLogConfig = {};
+configurable AccessLogConfiguration & readonly accessLogConfig = {};
+
+isolated function initializeHttpLogs() {
+    handle httpLogManager = newHttpLogManager(traceLogConfig, accessLogConfig);
```

Comment:
Do we use `httpLogManager` anywhere else? If not we can avoid the return value from the function

Reply:
`httpLogManager` is not used elsewhere. But the external function is a Java constructor which returns the object. That's why I assigned it to a value. Anyway to avoid this?

Reply:
We can change the extern function not to return anything.
https://github.com/ballerina-platform/module-ballerina-http/blob/f765707bb4d9a9d29c489df450c2b3b0575d91d3/ballerina/http_request.bal#L660